WordPress/Classes/Models/Blog.m

```diff
@@ -64,8 +68,8 @@ - (void)didTurnIntoFault
     [super didTurnIntoFault];
 
     // Clean up instance variables
-    _blavatarUrl = nil;
-    _api = nil;
+    self.blavatarUrl = nil;
+    self.api = nil;
```

Comment:
I'm not really sure how i managed to confuse a setter call, with a getter method. But in fact i did! sorry about that!